src/Core/src/Sts/Input/AssumeRoleRequest.php

```diff
@@ -22,6 +22,13 @@ final class AssumeRoleRequest extends Input
 
     /**
      * An identifier for the assumed role session.
+     * Use the role session name to uniquely identify a session when the same role is assumed by different principals or for
```

Comment:
I would prefer it if we could separate the first line from the rest with a blank line.

```
    /**
     * An identifier for the assumed role session.
     *
     * Use the role session name to uniquely identify a session when the same role is assumed by different principals or for
     *
```

In the original psr draft phpdoc spec, the first paragraph has special meaning as the summary description.

Reply:
This should be done by php-cs-fixer. 

They usually help us so we don’t need to add any logic for formatting.

Reply:
The GeneratorHelper removes empty lines in `$s = preg_replace('/\n+/', "\n", $s);`. Maybe removing this would be enough.